examples/src/main/java/org/apache/ignite/example/storage/VolatilePageMemoryStorageExample.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.example.storage;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * This example demonstrates a usage of the PageMemory storage engine configured with an in-memory data region.
+ *
+ * <p>To run the example, do the following:
+ * <ol>
+ *     <li>Import the examples project into you IDE.</li>
+ *     <li>
+ *         Download and prepare artifacts for run Ignite node using the CLI tool (if not done yet):<br>
+ *         {@code ignite init}
+ *     </li>
+ *     <li>
+ *         Add module "ignite-storage-page-memory" using the CLI tool (if not done yet):<br>
```

Comment:
Shouldn't it be there by default?

Reply:
@slukyano Do you think we should add the `PageMemory` storage engine to the **ignite-3** default jar or as an add-on module?

If we add, the user may get the following problems:
- It is necessary to explicitly specify the engine when creating tables through SQL, or do not forget to set up a cluster configuration where the engine will be set by default.

I suggest making the engine as addable.

Reply:
Pretty sure `PageMemory` should be the default. It preserves compatibility with 2.x behavior better, and it's more universal than LSM.